index.html

```diff
@@ -191,6 +191,7 @@ <h1 id="heading-card-3">Website Protfolio
     </div>
 </section>
 <section id="about-me">
+    <div id="grid-about-me"></div>
```

Comment:
- [ ] `required`
- [ ] Amazing so far 👏 
- [ ] please remove the dot style on the social button so that you can match the figma design.
- [ ] please see the screenshot below

![image](https://user-images.githubusercontent.com/66230306/199962546-217739d6-3c54-4d6a-9f79-877d783c0fa0.png)

Reply:
alright removed the dot from the list style